src/components/MapView/index.tsx

```diff
@@ -410,6 +412,24 @@ function MapView({ classes }: MapViewProps) {
     urlDate,
   ]);
 
+  const watchBoundaryChange = (map: Map) => {
+    const { setBounds, setLocation } = boundaryInfoStateSlice;
+    const onDragend = () => {
+      const bounds = map.getBounds();
+      dispatch(setBounds(bounds));
+    };
+    const onZoomend = () => {
+      const bounds = map.getBounds();
+      const newZoom = map.getZoom();
+      dispatch(setLocation({ bounds, zoom: newZoom }));
+    };
+    map.on('dragend', onDragend);
+    map.on('zoomend', onZoomend);
+    // Show initial value
+    onZoomend();
+  };
+
+  const showBoundaryInfo = process.env.REACT_APP_SHOW_BOUNDARY_INFO === 'true';
```

Comment:
Maybe use JSON.parse(REACT_APP_SHOW_BOUNDARY_INFO)?

https://stackoverflow.com/a/54906327/16585247

Please make sure it is False when the env is not set

Reply:
Resolved in [4051438](https://github.com/WFP-VAM/prism-frontend/pull/525/commits/40514389418d2fbdf6e42c4120638fab65292c7d)